pkg/apis/flowcontrol/types.go

```diff
@@ -265,6 +266,15 @@ type ResourcePolicyRule struct {
 	// +optional
 	// +listType=set
 	Namespaces []string
+
+	// `resourceNames` is a list of target resource names that restricts
+	// matches. A (get, update, patch, delete) request that specifies
+	// a target name matches if either (a) this list contains that target
+	// names or (b) this list contains "*".
```

Comment:
Look above at the comment on `ResourcePolicyRule`.  That needs to be extended, in the same style that it already has, and this field's comment needs to play its part.